static/src/stylesheets/module/content/_content.scss

```diff
@@ -173,6 +180,10 @@
 
 }
 
+.content__series-label--immersive-article--article {
```

Comment:
I'm confused by the double mod `--immersive-article--article`, why would you need this? The only reason I could see is if we were using something like `[class^="content__series-label"][class*="--immersive-article"] {...}` and  `[class^="content__series-label"][class*="--article"] {...}` to target styles (but we're not obviously). 

You probably were going for `--immersive-article-article` for whatever you're trying to do, but that's obviously not much clearer so I think maybe there's a re-think needed?